src/events/http/HttpServer.js

```diff
@@ -906,9 +906,11 @@ export default class HttpServer {
         const apiKey = createApiKey()
 
         this.#apiKeysValues = getApiKeysValues(
-          this.#serverless.service.provider.apiGateway?.apiKeys ?? [apiKey],
```

Comment:
This doesn't work correctly. The result of this expression is that `this.#apiKeysValues` is an empty set which means the key that was just created will not work.

Reply:
yes, you are right. the problem is that we are passing the result of the expression to `getApiKeysValues`, which expects an array of objects, while `[apiKey]` is just an array of a string.